### PR TITLE
Fix/recovery string

### LINKED ISSUE #12924: Suite shows you incorrect info during wallet recovery on T1
https://github.com/trezor/trezor-suite/assets/152851523/faa9a96f-355d-4404-893a-603120a54067

### PR DESCRIPTION
**Pull request by komret**
## Description

[This commit](https://github.com/trezor/trezor-suite/commit/e13edc518123754f540cfdb7d118999467c3ccbb#diff-8503efed95db087d858cdf351789d0d357d8d76bb775e42bf27e8a5fb6d4d070) broke it.

## Related Issue

Resolve https://github.com/trezor/trezor-suite/issues/12924

## Screenshots:
Screenshot 2024-07-02 at 19 25 29

### CODE REVIEW

File: packages/device-utils/tests/modelUtils.test.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+            description: 'should return null when the value for the picked device is null',
+            deviceModelInternal: DeviceModelInternal.T1B1,
+            expectedResult: null,
+        },
+        {
+            description: 'should return default when deviceModelInternal is undefined',
+            deviceModelInternal: undefined,
+            expectedResult: 'default',
+        },
+    ];
```
Comment (tomasklim) [inline packages/device-utils/tests/modelUtils.test.ts]: fixtures should be in different file

Comment (tomasklim) [reply packages/device-utils/tests/modelUtils.test.ts]: options can be part of fixtures, somehow

File: packages/device-utils/tests/modelUtils.test.ts [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        {
+            description: 'should return null when the value for the picked device is null',
+            deviceModelInternal: DeviceModelInternal.T1B1,
+            expectedResult: null,
+        },
+        {
+            description: 'should return default when deviceModelInternal is undefined',
+            deviceModelInternal: undefined,
+            expectedResult: 'default',
+        },
```
Comment (tomasklim) [inline packages/device-utils/tests/modelUtils.test.ts]: one case is not covered

```       
 {
            description: 'should return default when deviceModelInternal is not in options',
            deviceModelInternal: DeviceModelInternal.T2B1,
            expectedResult: 'default',
        },

        ```

**Review by tomasklim — CHANGES_REQUESTED** [review]

File: packages/device-utils/tests/modelUtils.test.ts (hunk 1 above)
Comment (komret) [reply packages/device-utils/tests/modelUtils.test.ts]: 18711f2b98c9e8493509db57009a79088ced546d

File: packages/device-utils/tests/modelUtils.test.ts (hunk 2 above)
Comment (komret) [reply packages/device-utils/tests/modelUtils.test.ts]: b0f77a6c2f5236c6e1982285d01d9598db784427

**Review by tomasklim — APPROVED** [review]

**Comment by STew790** [comment]
**QA OK**
image

**Info:**
 - Suite version: desktop 24.7.1 (88dbe0410ac5b6245f392ce13e8c30f0f8f549f2)
 - Browser: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) TrezorSuite/24.7.1 Chrome/118.0.5993.159 Electron/27.3.8 Safari/537.36
 - OS: Linux x86_64
 - Screen: 1920x1080
 - Device: Trezor T1B1 1.12.1 regular (revision 1eb0eb9d91b092e571aac63db4ebff2a07fd8a1f)
 - Transport: BridgeTransport 2.0.32

**Pull request merged**